packages/common/src/intl/date/IntlDate.ts

```diff
@@ -0,0 +1,9 @@
+import { useTranslation as useTranslationNext } from 'react-i18next';
+
+export const useFormatDate = (): ((
+  value: number | Date,
+  options?: Intl.DateTimeFormatOptions & { format?: string }
+) => string) => {
+  const { t } = useTranslationNext('app');
+  return (val, formatParams) => t('intl.datetime', { val, formatParams });
+};
```

Comment:
I just took the functions related to a specific thing and put it in a folder, basically..